### ISSUE TITLE
How to adjust AMD GPU frequency?

### ISSUE THREAD

**Issue by ysh329** [amd/OpenCL-caffe#65]
Maybe here's not a appropriate place to ask questions about AMD GPU, but I don't have other choice. I found wiki page [Manage your GPU HW · amd/OpenCL-caffe Wiki](https://github.com/amd/OpenCL-caffe/wiki/Manage-your-GPU-HW) referring the frequency adjustment but not detailed. 

What is AGT tools? I found a windows version this link ([Download AMD GPU Clock Tool | TechPowerUp](https://www.techpowerup.com/download/amd-gpu-clock-tool/) or [download from Guru3D.com](http://downloads.guru3d.com/AMD-GPU-Clock-Tool-v0.9.8-download-1985.html) but they're all Windows tools not for linux) using keyword AGT (AMD GPU tools).

Thus, I want to ask have any tools to adjust and query current GPU frequency? (I tried `aticonfig` but [failed](https://github.com/ysh329/OpenCL-101/issues/24)).

Thanks in advance!

**Comment by jlgreathouse** [comment]
You're right, this likely isn't the right place to ask this. Besides this being an experimental repo (and thus no longer supported or actively followed, as you might have noticed), the wiki entry you're referring to should likely have never been added.

AGT is an AMD-internal tool that is not released to the public, and thus the answer on that wiki page is not particularly useful.

That said, the [ROCm software stack](https://rocm.github.io/) includes a tool that allows you to both adjust and query AMD GPU frequencies. [ROCm-smi](https://github.com/RadeonOpenCompute/ROC-smi) is a tool that interfaces with the [ROCK kernel drivers](https://github.com/RadeonOpenCompute/ROCK-Kernel-Driver) to give you low-level monitoring and management access to AMD GPUs. ROCm-SMI should be installed on your system by default after installing ROCm (likely in /opt/rocm/bin/).
